src/components/Announce/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import strings from 'lang';
+import { connect } from 'react-redux';
+import { getGithubPulls } from 'actions/githubPulls';
+import ReactMarkdown from 'react-markdown';
+
+import styles from './styles.css';
+
+const firstPrio = 'blog';
+const secondPrio = 'ui';
+
+class RequestLayer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      repo: firstPrio,
+      dismissed: false,
+    };
+
+    this.dismiss = (date) => {
+      if (date && localStorage) {
+        localStorage.setItem('dismiss', date);
+        this.setState({
+          dismissed: true,
+        });
+      }
+    };
+  }
+
+  componentWillMount() {
+    this.props.getData(this.state.repo);
+  }
+
+  componentWillUpdate(nextProps, nextState) {
+    if (this.state.repo !== nextState.repo) {
+      this.props.getData(nextState.repo);
+    }
+  }
+
+  render() {
+    const { error, loading, data } = this.props;
+
+    if (!error && !loading && data && data.length) {
+      const {
+        merged_at: mergedAt,
+        title,
+        body,
+        html_url: link,
+      } = data[0];
+
+      if (mergedAt) {
+        // Fetch ui if blog's pr was merged more than 7 days ago
+        if (new Date().getDate() - new Date(mergedAt).getDate() >= 7 && this.state.repo === firstPrio) {
+          this.setState({ repo: secondPrio });
+        }
+
+        if (localStorage.getItem('dismiss') !== mergedAt) {
```

Comment:
maybe use repo name + pull request ID as a key?  It's unlikely but blog and releases could have the same merge time.